### PR TITLE
glTF Snapshot Tests

### PR DESCRIPTION
**Pull request by georgios-uber**

### CODE REVIEW

File: test/render/gltf-test-cases.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  const animationLoop = new AnimationLoop(app);
+  let count = 0;
+  return {
+    name,
+    onInitialize: params => {
+      return animationLoop.onInitialize(params);
+    },
+    onRender: params => {
+      animationLoop.onRender(params);
+      if (count++ > 2000) {
```
Comment (georgios-uber) [inline test/render/gltf-test-cases.js]: @Pessimistress I need a way to wait here till textures load...

Comment (Pessimistress) [reply test/render/gltf-test-cases.js]: - See the discussion at https://github.com/uber/luma.gl/pull/911#discussion_r257047484
- You can use `params.time` or `params.tick` for a hard-coded wait period

File: examples/gltf/app.js [hunk 2]
```diff
@@ -190,6 +194,11 @@ animationLoop.getInfo = () => INFO_HTML;
 
 export default animationLoop;
 
+// snapshot tests
+export const avocado = new DemoApp(GLTF_BASE_URL + "Avocado/glTF-Binary/Avocado.glb");
```
Comment (Pessimistress) [inline examples/gltf/app.js]: Export `DemoApp` instead and move the specific scenes into test/

File: scripts/test.sh [hunk 3]
```diff
@@ -63,6 +63,10 @@ case $MODE in
     NODE_ENV=production webpack --config test/webpack.config.js --env.import-nothing --env.es6
     break;;
 
+  "browser-headless")
```
Comment (Pessimistress) [inline scripts/test.sh]: `npm run test borwser-headless` already does this

File: test/render/gltf-test-cases.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        params.done();
+      }
+    },
+    onFinalize: animationLoop.onFinalize.bind(animationLoop),
+    goldenImage: `./test/render/golden-images/${name}.png`
```
Comment (Pessimistress) [inline test/render/gltf-test-cases.js]: The golden images should be in this PR

Comment (georgios-uber) [reply test/render/gltf-test-cases.js]: The problem is I am only getting empty images, not sure why.

File: scripts/test.sh (hunk 3 above)
Comment (georgios-uber) [reply scripts/test.sh]:
```
Running test suite in "borwser-headless" mode...
Unknown test mode borwser-headless

```

**Pull request merged**